### PR TITLE
Eslint fix task (beautify)

### PR DESCRIPTION
**Pull request by settinghead**

### CODE REVIEW

**Comment by settinghead** [comment]
@ezequielc @alex-deaconu Effect of the fix is demonstrated in this PR: https://github.com/Rise-Vision/viewer/pull/117

Please review. Thanks!

**Comment by alex-deaconu** [comment]
@settinghead I've reviewed the other PR and there are a couple of inconsistencies I've spotted. 

Also, the build is failing over in this PR, if you could take a look. Thanks.

**Comment by settinghead** [comment]
@alex-deaconu @ezequielc  I made a few small adjustment re. indentation, and the configuration should be complete. Thanks.

File: package.json [hunk 1]
```diff
@@ -48,9 +48,10 @@
     "chai": "^3.5.0",
     "chai-as-promised": "^5.3.0",
     "eslint": "^3.4.0",
-    "eslint-config-airbnb": "^10.0.1",
+    "eslint-config-airbnb": "^11.0.0",
     "eslint-plugin-import": "^1.14.0",
     "eslint-plugin-jsx-a11y": "^2.2.1",
+    "gulp-if": "^2.0.1",
```
Comment (alex-deaconu) [inline package.json]: @settinghead is this dependency used? Thanks.

**Comment by alex-deaconu** [comment]
@settinghead sounds good re disabling Coveralls notifications. They are annoying.

One comment in the other PR re missing files, I believe they were probably just deleted by mistake.

Also one comment here re: dependency that doesn't seem to be used.

Otherwise, looks good 👍 . Thanks!

**Comment by settinghead** [comment]
@alex-deaconu @ezequielc  removed unused dependency and confirmed that the file deletion was a human error. I'll be merging this later tonight if there are no other problems. Thanks!

File: package.json (hunk 1 above)
Comment (settinghead) [inline package.json]: Good catch. Not used. Removed.

**Pull request merged**